samples/client/petstore/kotlin-jvm-volley/src/main/java/org/openapitools/client/request/RequestFactory.kt

```diff
@@ -0,0 +1,87 @@
+// Knowing the details of an operation it will produce a call to a Volley Request constructor
+package org.openapitools.client.request
+
+
+import com.android.volley.Request
+import com.android.volley.Response
+import android.util.Base64
+import org.openapitools.client.request.IRequestFactory.Companion.escapeString
+import java.lang.reflect.Type
+import java.util.Locale
+import java.util.UUID
+
+class RequestFactory(private val headerFactories : List<() -> Map<String, String>> = listOf(), private val postProcessors :List <(Request<*>) -> Unit> = listOf(), private val gsonAdapters: Map<Type, Any> = mapOf()): IRequestFactory {
+
+   companion object Authentication {
+    // Where a header factory requires parameters a client will need to bind these
+    // TODO Generate appropriate header factories based on settings
+     // Api Key auth supports query header and cookie.
+     // Query is supported in the path generation only with a hardcoded value.
+     // TODO: Not sure about cookie auth form
+     // If implementing api key in query parameter use the ^isKeyInHeader property
+
+     val apiKeyAuthHeaderFactoryBuilder = {
+          paramName: String, apiKeyPrefix: String?, apiKey: String? -> {
+              mapOf("$paramName" to
```

Comment:
@alisters could you please replace `mapOf("$paramName" to` with `mapOf(paramName to`?

Reply:
See above

Reply:
Sorry, I didn't noticed that this was the generated code 👍